### PR TITLE
Add Embedded ES resource quota configuration UI

### PR DESCRIPTION
**Pull request by vimniky**
Related issue:

https://github.com/rancher/rancher/issues/11779

### CODE REVIEW

File: lib/logging/addon/components/logging/new-edit/component.js [hunk 1]
```diff
@@ -130,17 +130,18 @@ export default Ember.Component.extend(NewOrEdit, {
   }.property('originalModel.{id,targetType}', 'targetType'),
 
 
-  validate() {
```
Comment (westlywright) [inline lib/logging/addon/components/logging/new-edit/component.js]: if this isn't needed please remove

File: lib/logging/addon/components/logging/new-edit/template.hbs [hunk 2]
```diff
@@ -1,3 +1,4 @@
+{{log model}}
```
Comment (westlywright) [inline lib/logging/addon/components/logging/new-edit/template.hbs]: remove log

File: lib/logging/addon/components/logging/target-embedded/component.js [hunk 3]
```diff
@@ -1,4 +1,24 @@
 import Component from '@ember/component';
 import es from 'logging/mixins/target-elasticsearch';
+import { get, set } from '@ember/object';
 
-export default Component.extend(es, {});
+export default Component.extend(es, {
+
+  init(...args) {
```
Comment (westlywright) [inline lib/logging/addon/components/logging/target-embedded/component.js]: Init doesn't have to take `...args`, Ember convention is `this._super(...arguments)`

File: lib/logging/addon/mixins/logging-model.js [hunk 4]
```diff
@@ -22,14 +25,12 @@ export default Mixin.create({
...
     loggingTagets.forEach(key => {
       const config = store.createRecord({
-        type: `${key}Config`
+        type: `${key}Config`,
       });
-      const clone = nue.clone();
-      clone.set('config', config);
-      map[key] = clone;
+      nue.set('config', config);
+      map[key] = nue.clone();
```
Comment (westlywright) [inline lib/logging/addon/mixins/logging-model.js]: Can you update this to use Embers set method?  ```set(map, key, nue.clone())```

**Comment by vimniky** [comment]
Updated and I just rebased master!

**Pull request merged**